src/sentry/static/sentry/app/components/events/eventEntries.jsx

```diff
@@ -195,7 +195,12 @@ class EventEntries extends React.Component {
       <div className={className} data-test-id="event-entries">
         {!objectIsEmpty(event.errors) && (
           <ErrorContainer>
-            <EventErrors event={event} />
+            <EventErrors
+              event={event}
+              orgId={organization.slug}
+              project={project}
+              issueId={group.id}
```

Comment:
This is something for UI folks like @priscilawebdev or @matejminar. I think we have context adapters (like `withProjectContext` or similar), so that you can access the ambient project and group without having to pass it through. Not sure if that's desired here, though.